# Sends a command to play a predifined sond
# The available song yet are
# 0 - Star Wars
# 1 - Indiana Jones

uint8 Star_Wars			= 0
uint8 Indiana_Jones		= 1
uint8 Au_Clair_De_La_Lune	= 2
uint8 La_Marseillaise		= 3
uint8 Lavender_Town		= 4
uint8 J2			= 5
uint8 Happy_Birthday		= 6
uint8 Titanic			= 7

uint8 song
